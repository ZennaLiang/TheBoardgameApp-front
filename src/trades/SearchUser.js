
import React from "react";
import Button from "react-bootstrap/Button";

 class SearchUser extends React.Component {

    render() {

      return (
        <div>
        <input id='searchbar' className='w-25 mt-2 rounded' type='text' name='search' placeholder='Search...' value={this.props.searchValue} onChange={this.props.onChangeValue}/>
        <Button variant="primary" id="userInput">Search</Button></div>

      );
    }
  }
  export default SearchUser;