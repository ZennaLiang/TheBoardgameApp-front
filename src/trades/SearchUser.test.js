import React from "react";
import renderer from "react-test-renderer";
import SearchUser from "./SearchUser";

describe("<SearchUser />", () => {
  it("render SearchUser", () => {
    const tree = renderer.create(<SearchUser />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
