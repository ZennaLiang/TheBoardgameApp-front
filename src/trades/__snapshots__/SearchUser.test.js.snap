// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchUser /> render SearchUser 1`] = `
<div>
  <input
    className="w-25 mt-2 rounded"
    id="searchbar"
    name="search"
    placeholder="Search..."
    type="text"
  />
  <button
    className="btn btn-primary"
    disabled={false}
    id="userInput"
    type="button"
  >
    Search
  </button>
</div>
`;
